### PR TITLE
Data generation script 

### PR DESCRIPTION
**Pull request by ike-okonkwo**
**Summary**

This PR has some basic setup scripts,  data generation files and schema files

### CODE REVIEW

File: database/data.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    status : faker.random.number({"min" :0, "max" : 1})
+  };
+  return readStatus;
+};
+
+
+var dataWriteToDisk = function(fileName, batch, callback){
+
+  var dataToStore = [];
+  var noOfProfiles =  250000;  // generate 250K records per ass
```
Comment (Danielxburns) [inline database/data.js:51]: In your comment, did you mean 'pass'?

File: database/data.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+// generate data
+var data_bookInfo = function(){
+  var bookInfo = {
+    title : faker.random.words(3),
+    author : faker.name.findName(),
+    description : faker.lorem.sentences(6),
+    num_reviews : faker.random.number({"min" :1, "max" : 50000}),
+    num_ratings : faker.random.number({"min" :1, "max" : 2000000}),
+    ratings_star : faker.random.number(0, 5, 2),
+    image :faker.image.image()
```
Comment (Danielxburns) [inline database/data.js:24]: I see you have a space before your colons. What linter are you using? The airbnb one gives me grief if I do this.

**Review by Danielxburns — COMMENTED** [review]
This is very clean and well ogranized. Nice job, Ike!

**Review by jordisman — COMMENTED** [review]
I'm curious, is this setup.txt your notes for different databases setup?

File: database/data.js (hunk 2 above)
Comment (jordisman) [reply database/data.js:24]: I'm curious about the space too. I thought it would not work.

**Comment by therobinkim** [comment]
@ike-okonkwo as already mentioned by the others, I would recommend configuring eslint asap for Airbnb's style guide and fixing up your codebase as per its recommendations.

File: database/data.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
+var faker = require('faker');
+var csv = require('csv-parser');
+var fs = require('fs');
+
+//set seed
+faker.seed(123);
+
+var generateRating = function(max){
+  // return random Number between 0 and max
+  var randomNum = parseInt(Math.random() * max)
```
Comment (therobinkim) [inline database/data.js:10]: I believe this function returns a random number btwn 0 and max-1, not 0 and max.

File: database/data.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  return randomNum;
+};
+
+
+// generate data
+var data_bookInfo = function(){
```
Comment (therobinkim) [inline database/data.js:16]: I recommend more parallelism w/ your function names.

If you generate ratings with a function called `generateRating()`, perhaps you should generate book info with a function called `generateBookInfo()`.

File: package.json [hunk 5]
```diff
@@ -36,17 +37,22 @@
...
     "cors": "^2.8.5",
     "css-loader": "^2.1.1",
+    "csv-parser": "^2.2.0",
+    "dev": "^0.1.3",
     "dotenv": "^7.0.0",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.12.1",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
+    "fs": "0.0.1-security",
```
Comment (therobinkim) [inline package.json:47]: `fs` is built into Node.js and should not be installed separately.

File: database/data.js (hunk 1 above)
Comment (ike-okonkwo) [reply database/data.js:51]: Just  fixed that

File: database/data.js (hunk 2 above)
Comment (ike-okonkwo) [reply database/data.js:24]: Forgot to set up lint for this. Checking  through and fixing all issues

**Comment by ike-okonkwo** [comment]
> I'm curious, is this setup.txt your notes for different databases setup?

I'm just keeping track of my setup notes

File: database/data.js (hunk 3 above)
Comment (ike-okonkwo) [reply database/data.js:10]: I wasn't using this function so I just removed it

File: database/data.js (hunk 4 above)
Comment (ike-okonkwo) [reply database/data.js:16]: parsed through code and made function names more expressive ->  `generateBookInfo` , `generateUsers`,`generateReadStatus`

File: package.json (hunk 5 above)
Comment (ike-okonkwo) [reply package.json:47]: removed from `package.json`

**Comment by ike-okonkwo** [comment]
> @ike-okonkwo as already mentioned by the others, I would recommend configuring eslint asap for Airbnb's style guide and fixing up your codebase as per its recommendations.

Setup Airbnb lint and fixing associated lint issues

**Pull request merged**